backend/tests/integration/records/serializers/test_bibtex.py

```diff
@@ -139,3 +142,54 @@ def test_bibtex_search(api_client, db, es, create_record):
     assert expected_status_code == response_status_code
     assert expected_result_1 in response_data
     assert expected_result_2 in response_data
+
+
+def test_bibtex_doesnt_encode_math_environments(api_client, db, es, create_record):
+    headers = {"Accept": "application/x-bibtex"}
+    data = {
+        "control_number": 637275237,
+        "titles": [
+            {
+                "title": "Low-energy theorem for $\\gamma\\to 3\\pi$: $\Sigma$ surface terms against $\\pi a_1$-mixing"
```

Comment:
```suggestion
                "title": "Low-energy theorem for $\\gamma\\to 3\\pi$: Σ surface terms against $\\pi a_1$-mixing"
```
The test needs to assert that if you have a unicode character, it gets encoded properly:
```
In [1]: import latexcodec                                                       

In [2]: import codecs                                                           

In [3]: codecs.encode("Σ", "ulatex")                                            
Out[3]: '$\\Sigma$'
```
If that doesn't happen, there's a bug in your logic, which this test is meant to prevent.